CodeSniffer/Standards/Generic/Sniffs/WhiteSpace/LanguageConstructSpacingSniff.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Generic_Sniffs_WhiteSpace_LanguageConstructSpacingSniff.
+ *
+ * PHP version 5
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @author    Marc McIntyre <mmcintyre@squiz.net>
+ * @author    George Mponos <gmponos@gmail.com>
+ * @copyright 2006-2014 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ * @link      http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+/**
+ * Generic_Sniffs_WhiteSpace_LanguageConstructSpacingSniff.
+ *
+ * Ensures all language constructs (without brackets) contain a
+ * single space between themselves and their content.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @author    Marc McIntyre <mmcintyre@squiz.net>
+ * @author    George Mponos <gmponos@gmail.com>
+ * @copyright 2006-2014 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ * @version   Release: @package_version@
+ * @link      http://pear.php.net/package/PHP_CodeSniffer
+ */
+class Generic_Sniffs_WhiteSpace_LanguageConstructSpacingSniff implements PHP_CodeSniffer_Sniff
+{
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        return array(
+                T_ECHO,
+                T_PRINT,
+                T_RETURN,
+                T_INCLUDE,
```

Comment:
Not sure about `include`, `require`, `include_once` and `require_once` because they can be written in 2 ways:

* as keyword: `include_once 'file.php';`
* as function call: `include_once('file.php');`

In later case it will be auto-fixed as `include_once ('file.php');` which will trigger warning in another sniff checking function calls.